src/frontends/lean/token_table.cpp

```diff
@@ -96,7 +96,7 @@ void init_token_table(token_table & t) {
          {"@@", g_max_prec}, {"@", g_max_prec},
          {"sorry", g_max_prec}, {"+", g_plus_prec}, {"->", g_arrow_prec}, {"<-", 0},
          {"match", 0}, {"^.", g_max_prec+1},
-         {"renaming", 0}, {"extends", 0}, {nullptr, 0}};
+         {"renaming", 0}, {"extends", 0}, {"generalizing", 0}, {nullptr, 0}};
```

Comment:
You mean I should do
```lean
precedence `generalizing` : 0
```
instead of changing `token_table.cpp`? That's definitely better, right, but it would still be _even_ better if it was a keyword just in that tactic instead of globally.